src/test/java/no/sikt/nva/UnisContentScrapingResultTest.java

```diff
@@ -98,4 +99,46 @@ void shouldThrowExceptionIfUnknownColumnHeaderExists() {
                      () -> UnisContentScrapingResult.fromWorkbook(createWorkbook(invalidExcelData)),
                      ERROR_MESSAGE_INVALID_HEADERS);
     }
+
+    @Test
+    void shouldAppendContentFilesToSameRecordWhenCristinIdIsIdentical()
```

Comment:
split this test maybe?

Reply:
I can't see how this test does more than one thing? So I disagree with this comment.